### DISCUSSION TITLE
Is there any plan about  API document or open source the copilot extensions?

### DISCUSSION THREAD

**Discussion by bearguo** [community/community#61742] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

 Our organization is subscribing copilot business plan.  And we are wondering is there some way to re-build our own copilot extension base on copilot API so that we can get some specific suggest about our business situation.That will be great help for our coding.

**Comment by Akash1134** [comment]
Hi @bearguo ,

Thank you sharing your feedback. I have asked my team internally to gather more details about any plans to provide open source API for Copilot.
